internal/sapsystem/sapsystem.go

```diff
@@ -77,6 +78,26 @@ type CustomCommand func(name string, arg ...string) *exec.Cmd
 
 var customExecCommand CustomCommand = exec.Command
 
+func (sm SAPSystemsMap) GetSIDsString() string {
```

Comment:
This duplication looks horrible hehe
I'm sure you will be able to remove this 1st function once you have remove the `.Host.GetSIDstring`